apps/vs-code-designer/src/app/utils/codeless/startDesignTimeApi.ts

```diff
@@ -178,3 +194,34 @@ export function stopDesignTimeApi(): void {
   }
   ext.workflowDesignChildProcess = undefined;
 }
+
+export async function promptStartDesignTimeOption(context: IActionContext) {
+  if (vscode.workspace.workspaceFolders && vscode.workspace.workspaceFolders.length > 0) {
+    const workspace = await getWorkspaceFolder(context);
+    const projectPath = await tryGetFunctionProjectRoot(context, workspace);
+    const autoStartDesignTimeKey = 'autoStartDesignTime';
+    const autoStartDesignTime = !!getWorkspaceSetting<boolean>(autoStartDesignTimeKey);
+    const showStartDesignTimeWarningKey = 'showStartDesignTimeWarning';
+    const showStartDesignTimeWarning = !!getWorkspaceSetting<boolean>(showStartDesignTimeWarningKey);
+    if (projectPath) {
+      if (autoStartDesignTime) {
+        startDesignTimeApi(projectPath);
+      } else if (showStartDesignTimeWarning) {
+        const message = localize('startDesignTimeApi', 'Always start design time on launch?');
```

Comment:
@ecfan also these messages  for the warning message

Reply:
```suggestion
        const message = localize('startDesignTimeApi', 'Always start workflow designer at launch?');
```

Reply:
@ecfan It doesn't open the designer at launch right now. Just starts the background processes to open the designer faster when they do.

@ccastrotrejo @nidhi-pathak  I believe we discussed that if the user only had one workflow in their project, we would open it for them. Thoughts? This can be looked at after this PR

Reply:
Customers might not understand that technical distinction. I'll revise the wording, but we should likely clarify.